test/to_string_test.hpp

```diff
@@ -0,0 +1,9 @@
+#include <iostream>
+using namespace std;
+void toStringTest()
+{
```

Comment:
https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions

```suggestion
void toStringTest() {
```